ql/src/test/results/clientpositive/llap/materialized_view_rewrite_window.q.out

```diff
@@ -1047,6 +1047,7 @@ PREHOOK: query: select
   program
 from tv_view_data
 PREHOOK: type: QUERY
+PREHOOK: Input: arc_view@mv_tv_view_data_av2
```

Comment:
No this test was added for testing calcite based rewrite. The text based rewrite modified the plan because I forgot to turn it off in this test case.
Turned off and reverted the out.